imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.kt

```diff
@@ -22,11 +22,13 @@ class ImagePickerConfig(
     var isOnlyVideo: Boolean = false,
     var isIncludeAnimation: Boolean = false,
     var isShowCamera: Boolean = true,
+    var isShowVideo: Boolean = true,
```

Comment:
1. "camcorder" or "video camera" is more accurate
2. Let's make this `false` by default.